co-simulation tool/docker/ns3-docker/user-guide-docker.md

```diff
@@ -0,0 +1,222 @@
+#  Running CARLA-SUMO-MOSAIC Co-Simulation Tool with NS-3 Integration in Docker 
```

Comment:
It is better to change the name of this user guide to README.md then Github will directly show the content in the page.

Reply:
The user guide has been renamed as README.md.